src/lib/dialog/dialog.scss

```diff
@@ -35,6 +35,9 @@ $mat-dialog-button-margin: 8px !default;
   max-height: $mat-dialog-max-height;
   overflow: auto;
   -webkit-overflow-scrolling: touch;
+
+  // Promote the content to a new GPU layer to avoid repaints on scroll.
+  backface-visibility: hidden;
```

Comment:
Why was it repainting on scroll in the first place?

Reply:
Better solution to moving to GPU is `transform: translate3d(0, 0, 0)`
Or the `will-change` property is getting more support

Reply:
@Lindsor the result is exactly the same. I went with `backface-visibility`, because `transform` can cause the content to be blurry in some browsers.

@jelbourn I'm not sure, it's all up to the browser. My best guess is because it's in a fixed element and it has an explicit `overflow: auto`.